omero/sysadmins/unix/ciphers

```diff
@@ -2,4 +2,4 @@ Weaker ciphers like ADH are disabled by default in new versions of OpenSSL
 and TLS versions 1.0 and 1.1 have been dropped from JDK packages.
 In order to connect to an OMERO.server using any OMERO clients
 e.g. the Java Desktop client, the OMERO.web client or the CLI and import data, 
-you need to generate self-signed certificates after installing the `omero-certificates <https://github.com/ome/omero-certificates>`_ package.
+you need to generate self-signed certificates using `omero-certificates <https://github.com/ome/omero-certificates>`_ package.
```

Comment:
Btw reason for this change is that omero-certificates is already installed (why, I don't know).